tests/unit_tests/constraint/test_fcialg.py

```diff
@@ -644,7 +668,13 @@ def test_fci_complex(self):
         context = make_context().variables(data=sample).build()
         oracle = Oracle(G)
         ci_estimator = oracle
-        fci = FCI(ci_estimator=ci_estimator, max_iter=np.inf)
+        fci = FCI(
+            ci_estimator=ci_estimator,
+            max_iter=np.inf,
+            skeleton_method=skeleton_method,
+            pds_skeleton_method=pds_skeleton_method,
+            selection_bias=False,
```

Comment:
```suggestion
            selection_bias=selection_bias,
```

Then perhaps we can add an extra parametrization for `selection_bias`. Because regardless, the graph in the RFCI paper does not have selection bias, so the relevant rules should not be activated.